pkgs/firedragon/update.nix

```
@@ -43,7 +43,7 @@ writeShellScript "update-firedragon" ''
   # FireDragon doesn't expose this information, instead writing its own version to this file
   firefoxVersion=$(curl -s https://raw.githubusercontent.com/Floorp-Projects/Floorp/ESR128/browser/config/version.txt)
 
-  latestSha256=$(nix-prefetch-url --type sha256 "https://gitlab.com/api/v4/projects/55893651/packages/generic/firedragon/$latestVer/firedragon-v$latestVer.source.tar.zst")
+  latestSha256=$(nix-prefetch-url --type sha256 "https://gitlab.com/api/v4/projects/69949446/packages/generic/firedragon/v$latestVer/firedragon-source-v$latestVer.tar.zst")
```

I suggest going through the release since this will be guaranteed to be static:

```suggestion
  latestSha256=$(nix-prefetch-url --type sha256 "https://gitlab.com/garuda-linux/firedragon/firedragon12/-/releases/v${latestVer}/downloads/firedragon-source.tar.zst")
```